CondCore/ESSources/test/stubs/TestUpdater.cc

```diff
@@ -21,19 +21,21 @@ EVENTSETUP_RECORD_REG(OneIntRcd);
 
 namespace condtest {
 
-  class TestUpdater : public edm::EDAnalyzer {
+  class TestUpdater : public edm::one::EDAnalyzer<> {
```

Comment:
```suggestion
  class TestUpdater : public edm::one::EDAnalyzer<edm::one::WatchRuns> {
```
this needs to watch runs, as it has a beginRun method (that should be marked as override)